packages/core/src/transport/eventBridge.ts

```diff
@@ -32,6 +32,11 @@ export function getEventBridge<T, E>() {
   }
 }
 
+export function isBridgeForRecordsSupported(): boolean {
+  const bridge = getEventBridgeGlobal()
+  return !!bridge && 'getPrivacyLevel' in bridge
```

Comment:
**❓ question:** ‏Did we discuss with mobile if it could be worth having a proper solution for that? Ex: exposing a Mobile SDK version, or "capabilities"?
I'm fine with keeping this in that PR / while on beta, but it would be great to have a better approach

Reply:
The idea would be for mobile to provide a method like `bridgeSupports('feature_name')`?
No it has not been discussed yet, but it is true that it could be nice to anticipate potential future breaking changes.

Reply:
Yes, or `getCapablities().includes("feature_name")`.